client/payment-details/dispute-details/dispute-awaiting-response-details.tsx

```diff
@@ -34,22 +35,29 @@ import { getAdminUrl } from 'wcpay/utils';
 import DisputeNotice from './dispute-notice';
 import IssuerEvidenceList from './evidence-list';
 import DisputeSummaryRow from './dispute-summary-row';
+import DisputeSteps from './dispute-steps';
 import InlineNotice from 'components/inline-notice';
 import './style.scss';
 
 interface Props {
 	dispute: Dispute;
+	customer: ChargeBillingDetails | null;
+	chargeCreated: number;
 }
 
-const DisputeAwaitingResponseDetails: React.FC< Props > = ( { dispute } ) => {
+const DisputeAwaitingResponseDetails: React.FC< Props > = ( {
+	dispute,
+	customer,
+	chargeCreated,
+} ) => {
 	const { doAccept, isLoading } = useDisputeAccept( dispute );
 	const [ isModalOpen, setModalOpen ] = useState( false );
 
 	const now = moment();
 	const dueBy = moment.unix( dispute.evidence_details?.due_by ?? 0 );
 	const countdownDays = Math.floor( dueBy.diff( now, 'days', true ) );
 	const hasStagedEvidence = dispute.evidence_details?.has_evidence;
-	const showDisputeActions = ! isInquiry( dispute );
+	const showDisputeStepsAndActions = ! isInquiry( dispute );
```

Comment:
Just noting this is temporary – a comment might be helpful here (i.e. this should get deleted when inquiry version is done).

Reply:
```suggestion
	// This is a temporary restriction and can be removed once steps and actions for inquiries are implemented.
	const showDisputeStepsAndActions = ! isInquiry( dispute );
```